README.md

```diff
@@ -112,7 +115,19 @@ Write a conditional statement that...
 8.  Write your javascript solution below
 ```js
 // your answer here
-```
+
+var num = []
+for (var i = 0; i <= 100; i++) {
+if (num % 3 === 0) {
+console.log("Fizz")
+}
+if (num % 5 === 0) {
+console.log("buzz")
+}
+if (num % 5 === 0 && num % 3 === 0  
```

Comment:
`if (num % 5 === 0 && num % 3 === 0  ` is missing `)`
also this should be the first condition and use else after it.